docs/failover-cluster.md

```diff
@@ -108,14 +105,14 @@ This new setup shares one drawback with the first one: if `scdata` crashes, the
 
 ***Note**: It is important to specify the database path using exactly the same value in all places where it occurs. Values such as `C:\Path\To\Db` & `C:/Path/To/Db` are treated as different.*
 
-*See also the article about the [Database connection string](database-connection-string.md#the-database-value).*
+_See also the article about the [Database connection string](database-connection-string.md#the-database-value)._
 
-#### 1. Setup cluster and CSV
+#### 1\. Setup cluster and CSV
```

Comment:
What does the `\` stand for here?

Reply:
Not sure, it's something that my editor adds to numbered lists in markdown headings because it's apparently "the right thing to do".

Reply:
Okay, we would have to validate that GitBook can swallow such syntax properly.

Reply:
I can do that. It's good to constantly check output while writing. Any way I can test?

Reply:
Do you have access to Starcounter GitBook? If you don't - you can always have your own account on GitBook and see how they render certain markdown.